packages/trader/src/Stores/Modules/Contract/contract-replay-store.js

```diff
@@ -163,8 +164,22 @@ export default class ContractReplayStore extends BaseStore {
         this.chart_state = state;
 
         switch (state) {
+            case 'INITIAL':
+                this.is_chart_fitting = false;
+                break;
+            case 'READY':
+                setTimeout(
+                    action(() => (this.is_chart_fitting = true)),
+                    100
+                );
+                break;
             case 'SCROLL_TO_LEFT':
-                this.is_chart_loading = false;
+                setTimeout(
+                    action(() => {
+                        this.is_chart_loading = false;
+                    }),
+                    900
```

Comment:
Could we have a comment to reflect the reason for timeouts?

Reply:
@salar-fs just address that